common.mk

```diff
@@ -9,7 +9,12 @@ install-glide:
 install-ci:
 	make install-vendor
 
-install-metalinter:
+install-metalinter-internal:
```

Comment:
Even with this, users will have to specify the linter in their json config as well. Might be better to easier to have them as separate build targets and let people opt in?

Reply:
I'm not quite following. Are you saying that because not everyone will use this linter, we should make the installation opt-in as well?

something like `install-metalinter-with-maptime` or just `install-maptime` ? Seems a little annoying to have to change your build to include new linters. Might as well not share the installation at that point right?

Reply:
I mean we shouldn't be installing things when people don't want them. Can keep the two separate: `install-metalinter` & `install-linter-maptime`. And add a third `install-metalinter-with-plugins` if you feel strongly about it. 

Don't want the default behaviour be to include things that might not be used.

Reply:
Makes sense, I made the change